.travis.yml

```diff
@@ -8,16 +8,19 @@ php:
   - 5.5
   - 5.6
   - 7.0
+  - 7.1
   - hhvm
 
 matrix:
   include:
     - php: 5.6
-      env: SYMFONY_VERSION="2.7.*"
+      env: SYMFONY_VERSION="2.7.*" SYMFONY_DEPRECATIONS_HELPER="weak"
     - php: 5.6
-      env: SYMFONY_VERSION="2.8.*" SYMFONY_DEPRECATIONS_HELPER="strict"
+      env: SYMFONY_VERSION="2.8.*"
     - php: 5.6
-      env: SYMFONY_VERSION="3.0.*" SYMFONY_DEPRECATIONS_HELPER="strict"
+      env: SYMFONY_VERSION="3.0.*"
+    - php: 7.1
+      env: SYMFONY_VERSION="4.0.0-BETA3"
```

Comment:
To use latest BETA (today 4.0.0-BETA4) version :
`env: DEPENDENCIES=beta`

and `before_install` :

```
- if [ "$DEPENDENCIES" = "beta" ]; then composer config minimum-stability beta; fi;
```